zipline/assets/assets.py

```diff
@@ -678,3 +730,113 @@ class AssetConvertible(with_metaclass(ABCMeta)):
 
 class NotAssetConvertible(ValueError):
     pass
+
+
+class AssetFinderCachedEquities(AssetFinder):
```

Comment:
Do you think we should name this with something more specific to Equities, since the behavior it adds to the base AssetFinder doesn't apply to all Assets?